src/routes/Birthday.ts

```diff
@@ -34,9 +46,11 @@ router.use('set', async ({ user, params }) => {
 });
 
 const helpHandler : Handler = async () => {
-  let message = '**Krijg elke ochtend een melding als iemand jarig is**\nHier volgen alle commando\'s voor verjaardagen';
-  message += '\n`ei bday show-all`: Laat alle geregistreerde verjaardagen zien';
-  message += '\n`ei bday set <DD/MM/YYYY>`: Registreerd jouw verjaardag';
+  let message = '**Krijg elke ochtend een melding als iemand jarig is**';
+  message += '\n`ei bday set <DD/MM/YYYY>`: Stel je geboortedatum in';
+  message += '\n`ei bday show-all`: Laat iedereens geboortedatum zien';
+  message += '\n`ei bday show-age`: Laat iedereens leeftijd zien';
+  message += '\n`ei bday <@user>`: Laat de geboortedatum en leeftijd van een user zien';
```

Comment:
Dit zou eventueel ook met template strings kunnen.

```js
let message = 'My `cool` message'
message += '\nMy `cooler` message'
console.log(message)
// >> "My `cool` message
//    My `cooler` message"
```

Is hetzelfde als;

```js
let message = `My \`cool\` message
My \`cooler\` message`
console.log(message)
// >> "My `cool` message
//    My `cooler` message"
```

Reply:
Of wat dacht je van Array.prototype.join?

```js
const options = ['I can do `this`', 'I can do `this` as well']
options.join('\n')
// >> "I can do `this`
//    I can do `this` as well"
```

Reply:
```ts
let message = `My \`cool\` message
My \`cooler\` message`
console.log(message)
// >> "My `cool` message
//    My `cooler` message"
```

Gaat de linter door schreeuwen vgm, heb niet getest met template strings

Het in een array gooien zou wel een oplossing zijn ja, hoef je ook niet meer de `\n` te schrijven